src/EditorFeatures/Core/ExternalAccess/VSTypeScript/Api/VSTypeScriptEditorFormattingServiceWrapper.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
nit: This is the wrong copyright now, but I don't know if there's some overarching plan to do passes to fix

Reply:
Good point.